packages/crypto/test/hmac.node.test.ts

```diff
@@ -1,10 +1,20 @@
+import type { BytesLike } from '@fuel-ts/interfaces';
 import { arrayify } from '@fuel-ts/utils';
 
-import { computeHmac } from '..';
+import { computeHmac as computeHmacNode } from '..';
+
+interface ComputeHmacNode {
+  (algorithm: 'sha256' | 'sha512', key: string, data: string): string;
+  register(func: (algorithm: 'sha256' | 'sha512', key: string, data: string) => BytesLike): void;
+  lock(): void;
+}
+
 /**
  * @group node
  */
 describe('computeHmac node', () => {
+  const computeHmac = computeHmacNode as ComputeHmacNode;
```

Comment:
Seems like we need to update some types from the CryptoApi

Reply:
I did start to look into this and it involves having dynamic types for `browser` and `node`. I wondered whether this complexity is even worth providing, given we don't actually use the functionality provided on the node side.

Reply:
Agree here @petertonysmith94 since we are only using the node types

Reply:
https://github.com/FuelLabs/fuels-ts/issues/3074